.github/workflows/mtags-auto-release.yml

```
@@ -15,7 +15,7 @@ on:
         # If you update this line after release
         #   just put the tag name (`v*.*.*`) here as in `metals_version.value` above.
         # Don't be confused if this value contains `*.*.*_mtags_release`
-        default: "v0.11.8"
+        default: "0.11.8_mtags_release"
```

```suggestion
        default: "v0.11.8_mtags_release"
```

Is this fine without `v` ?

---

yep - I already created it without v because it's a branch - https://github.com/scalameta/metals/tree/0.11.8_mtags_release